### DISCUSSION TITLE
Going back to same scroll position

### DISCUSSION THREAD

**Discussion by NicolasRannou** [falinsky/tmdb-viewer#15] [Q&A]
Hi, 

First of all great demo, very cool stuff!

I'm working on a similar project and I can't pinpoint how you manage to go back to the previous scroll position in the list, when you scroll in the list, click a card, then go back to the list at the exact same position.

Is there a setting somewhere you have to enable or does it just work out of the box for you?

Any feedback would be greatly appreciated!

Thanks!
Nicolas

**Comment by falinsky** [comment]
Hi Nicolas, thanks.

Regarding the behavior - it worked just out of the box for me from the beginning. I believe the main player here is the default browser behavior when the scroll of the window is used. Pay attention that I use [WindowScroller](https://github.com/bvaughn/react-virtualized/blob/master/docs/WindowScroller.md) component.
